README.md

```diff
@@ -1,14 +1,15 @@
-# syntax-augmentation-nmt
+# TreeSwap
 
 [![Tests](https://github.com/attilanagy234/syntax-augmentation-nmt/actions/workflows/run-tests.yaml/badge.svg)](https://github.com/attilanagy234/syntax-augmentation-nmt/actions/workflows/run-tests.yaml)
 
-Complimentary code for our paper [Syntax-based data augmentation for Hungarian-English machine translation](https://arxiv.org/abs/2201.06876) submitted to the XVIII. Conference on Hungarian Computational Linguistics.
+Complimentary code for our papers [Syntax-based data augmentation for Hungarian-English machine translation](https://arxiv.org/abs/2201.06876) and [Data Augmentation for Machine Translation via Dependency Subtree Swapping](https://arxiv.org/abs/2307.07025) submitted to the XVIII. and XIX.  Conference on Hungarian Computational Linguistics.
 
 ## Trained models
-You can download the trained models reported in our paper from [this link](https://drive.google.com/drive/folders/13t9M0-j96ylqtkH0nd422wVzHmkAjvhk).
+Our trained models are available on the HuggingFace Model Hub with usage steps:
+- [HU-EN](https://huggingface.co/SZTAKI-HLT/opennmt-hu-en)
+- [EN-HU](https://huggingface.co/SZTAKI-HLT/opennmt-en-hu)
```

Comment:
This is a little confusing, because we didn't publish models for the recent paper, just for hu-en as part of a previous paper. What I would do is:
- put this `Trained models` section a bit further down in the readme (above citation for example) and mention that these are the models from the paper `Syntax-based data augmentation for Hungarian-English machine translation` for hu-en and en-hu specifically